Nodejs/Setup/NodejsTools/NodejsTools.wxs

```diff
@@ -15,6 +15,7 @@
             <Directory Id="Dir_$(var.key)TestFrameworks" Name="TestFrameworks">
                 <Directory Id="Dir_$(var.key)TestFrameworksExportRunner" Name="ExportRunner" />
                 <Directory Id="Dir_$(var.key)TestFrameworksMocha" Name="Mocha" />
+                <Directory Id="Dir_$(var.key)TestFrameworksTape" Name="Tape" />
```

Comment:
The directory is defined, but it's not being used anywhere. You'll also need to include a File Include in NodejsToolsFiles.proj
```
       <File Include="TestFrameworks\tape\tape.js">
            <InstallDirectory>TestFrameworksTape</InstallDirectory>
        </File>
```